controller/contour/deployment_test.go

```diff
@@ -16,6 +16,8 @@ package contour
 import (
 	"testing"
 
+	oputil "github.com/projectcontour/contour-operator/util"
+
```

Comment:
nit: remove space

Reply:
@stevesloka imports for the project have been organized as such:

1. standard libs
2. local libs
3. non-std libs